pkg/sentry/socket/netstack/stack.go

```diff
@@ -467,7 +467,7 @@ func (s *Stack) RestoreCleanupEndpoints(es []stack.TransportEndpoint) {
 // SetForwarding implements inet.Stack.SetForwarding.
 func (s *Stack) SetForwarding(protocol tcpip.NetworkProtocolNumber, enable bool) error {
 	if err := s.Stack.SetForwardingDefaultAndAllNICs(protocol, enable); err != nil {
-		return fmt.Errorf("SetForwardingDefaultAndAllNICs(%d, %t): %s", protocol, enable, err)
```

Comment:
could we please revert all the %s -> %v changes?

Reply:
Do you mean that %s is supposed to be used for `tcpip.Error`?
I found %v is used for `tcpip.Error` on master branch.
https://github.com/google/gvisor/pull/7285/files/43d8bdc0ac644f0c651f74ccbbc8265472b11edd#diff-79a83f33f2f92217b4a4a39e5f7d39b550abe60c428a6bdecd6cd6b411d9b67aL244

Reply:
Yes, we're not perfectly consistent. %v cannot be checked by `go vet`, so let's stick with %s where it can be used without loss of functionality.

Reply:
I fixed the areas you pointed out because the inconsistencies are spread throughout this project.